src/gui/FileBrowser.cpp

```diff
@@ -326,6 +328,32 @@ FileBrowserTreeWidget::FileBrowserTreeWidget(QWidget * parent ) :
 
 }
 
+// This method returns a QList with paths (QString's) of all directories
```

Comment:
I'd add those two lines into the header (above the function). If you use `//!` there instead of `//`, it shows up in our doxygen documentation.